sharding/mock/enableEpochsHandlerMock.go

```diff
@@ -389,6 +389,16 @@ func (mock *EnableEpochsHandlerMock) IsMiniBlockPartialExecutionFlagEnabled() bo
 	return false
 }
 
+// IsManagedCryptoAPIsFlagEnabled returns false
```

Comment:
Question: why do we have this mock in the sharding package? Why not moving this in the testscommon package instead (can be done in a new PR) as it might be used elsewhere.

Reply:
we already have the stub in testscommon, but using it into nodesCoordinator package was leading to cyclic imports.

I decided that creating a new mock(altought this means duplicate code), with the needed functionality for this package was a smaller change than moving all the tests from package nodesCoordinator to a new nodesCoordinator_test package, as they directly use a lot of unexported methods